scripts/rv_expression_association/igll5_vep.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
+
+import hail as hl
+import numpy as np
+import pandas as pd
+from hail.methods import export_plink
+from cpg_utils.hail_batch import dataset_path, init_batch, output_path #, reference_path
+
+# object containing variants within a 50K window on either side of the IGLL5 gene
+MT = dataset_path('v0/IGLL5_50K_window.mt')
```

Comment:
Files tend to be grouped by type, then versioned. I would've have imagined something like `IGLL5/v0/<files>`. Not exactly part of this PR because you're reading it, but just noting.

Reply:
Hi Michael, sorry I made this a normal PR instead of a draft one by mistake! That said I think I now got it to work using notebooks / test objects and analysis-runner, so hopefully it's almost ready to review (I have also added a README).

As for the files structure I appreciate that there isn't much of one at the moment, again I was still very much in exploratory / testing phase.. what is the easiest way to re-arrange files? And also best practice when switching to main, presumably we want the same hierarchy if possible?